modules/javafx.graphics/src/main/native-prism-d3d/D3DPhongMaterial.cc

```diff
@@ -68,35 +67,41 @@ float * D3DPhongMaterial::getSpecularColor() {
 }
 
 bool D3DPhongMaterial::isBumpMap() {
-    return map[BUMP] ? true : false;
+    return map[map_type::bump] ? true : false;
```

Comment:
Since `map` contains pointers, you might consider `return map[map_type::bump] != NULL`.